yarn.lock

```diff
@@ -2,186 +2,132 @@
 # yarn lockfile v1
 
 
+"@ampproject/remapping@^2.2.0":
+  version "2.3.0"
+  resolved "https://global.block-artifacts.com/artifactory/api/npm/square-npm/@ampproject/remapping/-/remapping-2.3.0.tgz#ed441b6fa600072520ce18b43d2c8cc8caecc7f4"
```

Comment:
Thanks, @lkvposs! This PR seems to have replaced the lock file artifact URLs with a proxy that seems to be internal to block, so it work in the environment that's outside of Block. 
```
sergey@Sergeys-MacBook-Pro firewalker % ping global.block-artifacts.com
PING artifactory-proxy-nlb-23fb44370c0a303c.elb.ap-southeast-2.amazonaws.com (10.176.5.165): 56 data bytes
Request timeout for icmp_seq 0
Request timeout for icmp_seq 1
Request timeout for icmp_seq 2
Request timeout for icmp_seq 3
```